bitnami/postgresql-ha/Chart.yaml

```diff
@@ -27,4 +27,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 7.5.4
+version: 7.6.0
```

Comment:
Please fix the "no new line at end of the file" warning. You should have your IDE configured to add it automatically.